modules/shared/src/main/scala/org/tessellation/schema/transaction.scala

```diff
@@ -82,7 +92,7 @@ object transaction {
           Seq(
             source.coerce,
             destination.coerce,
-            amount.coerce.value.toString(16),
+            amount.coerce.value.toHexString,
```

Comment:
@TheMMaciek Please confirm it's correct

Reply:
anyway, I think it is:
```scala
scala> 2L.toHexString
val res0: String = 2

scala> BigInt(2L).toString(16)
val res1: String = 2

scala> Long.MaxValue.toHexString
val res2: String = 7fffffffffffffff

scala> BigInt(Long.MaxValue).toString(16)
val res3: String = 7fffffffffffffff
```

Reply:
I guess. If it gives the same results. Maybe the best would be to look at the implementation and if they overlap somewhere underneath.